hybrasyl/server
# Updating Example XMLs

## Description
Some of the XML elements / attributes have changed, so the Example XML files are currently showing errors in the server logs. These corrections update the XML files to remove these errors from occurring.

## Related PRs

## Checklist

## How to QA
- Replace XML files, run server. No errors should be indicated.
- Login with a character, ExampleInn has an Inn Keeper showing.

## Impacted Areas in Hybrasyl
- Example contents
